dll/3rdparty/libxslt/CMakeLists.txt

```diff
@@ -47,9 +43,10 @@ if(MSVC)
     add_importlibs(libxslt ntdll)
 endif()
 
+if ((NOT MSVC) OR USE_CLANG_CL)
```

Comment:
```suggestion
if(GCC OR CLANG)
```

Reply:
Are we setting the CLANG variable somewhere ? I don't see this documented here:
https://cmake.org/cmake/help/v3.17/manual/cmake-variables.7.html

Reply:
Yeah, we're setting `GCC` and `CLANG` https://github.com/reactos/reactos/blob/master/sdk/cmake/config.cmake#L39-L48